tensorflow/g3doc/api_docs/python/train.md

```diff
@@ -1332,6 +1332,7 @@ Wait till the Coordinator is told to stop.
 ### `class tf.train.QueueRunner` {#QueueRunner}
 
 Holds a list of enqueue operations for a queue, each to be run in a thread.
+It asynchronously runs the enqueue ops in different threads when the queuue is not full.
```

Comment:
Confirmed.

Is it obvious from the comment that enqueue operations run multiple times in a tight loop? Perhaps would be worth adding that?